app/src/main/res/layout/image_region_selection_interaction_item.xml

```diff
@@ -13,23 +13,39 @@
 
   <FrameLayout
     android:id="@+id/interaction_container_frame_layout"
-    android:layout_width="wrap_content"
+    android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="28dp"
-    android:layout_marginTop="@dimen/divider_margin_top"
-    android:layout_marginEnd="28dp"
-    android:layout_marginBottom="@dimen/divider_margin_bottom"
     android:background="@drawable/drag_drop_white_background"
     android:descendantFocusability="beforeDescendants"
     android:focusableInTouchMode="true"
-    android:paddingStart="8dp"
-    android:paddingTop="8dp"
-    android:paddingEnd="8dp"
-    android:paddingBottom="8dp">
+    android:paddingStart="16dp"
+    android:paddingTop="12dp"
+    android:paddingEnd="16dp"
+    android:paddingBottom="12dp"
+    app:explorationSplitViewMarginApplicable="@{viewModel.hasConversationView &amp;&amp; viewModel.splitView}"
+    app:explorationSplitViewMarginBottom="@{@dimen/space_0dp}"
+    app:explorationSplitViewMarginEnd="@{@dimen/interaction_item_split_view_margin_end}"
+    app:explorationSplitViewMarginStart="@{@dimen/interaction_item_split_view_margin_start}"
+    app:explorationSplitViewMarginTop="@{@dimen/interaction_item_split_view_margin_top}"
+    app:explorationViewMarginApplicable="@{viewModel.hasConversationView &amp;&amp; !viewModel.splitView}"
+    app:explorationViewMarginBottom="@{@dimen/space_0dp}"
+    app:explorationViewMarginEnd="@{@dimen/interaction_item_exploration_view_margin_end}"
+    app:explorationViewMarginStart="@{@dimen/interaction_item_exploration_view_margin_start}"
+    app:explorationViewMarginTop="@{@dimen/interaction_item_exploration_view_margin_top}"
+    app:questionSplitViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; viewModel.splitView}"
+    app:questionSplitViewMarginBottom="@{@dimen/space_0dp}"
+    app:questionSplitViewMarginEnd="@{@dimen/interaction_item_split_view_margin_end}"
+    app:questionSplitViewMarginStart="@{@dimen/interaction_item_split_view_margin_start}"
+    app:questionSplitViewMarginTop="@{@dimen/interaction_item_split_view_margin_top}"
+    app:questionViewMarginApplicable="@{!viewModel.hasConversationView &amp;&amp; !viewModel.splitView}"
+    app:questionViewMarginBottom="@{@dimen/space_0dp}"
+    app:questionViewMarginEnd="@{@dimen/interaction_item_question_view_margin_end}"
+    app:questionViewMarginStart="@{@dimen/interaction_item_question_view_margin_start}"
+    app:questionViewMarginTop="@{@dimen/interaction_item_question_view_margin_top}">
 
     <org.oppia.android.app.player.state.ImageRegionSelectionInteractionView
       android:id="@+id/image_click_interaction_image_view"
-      android:layout_width="wrap_content"
+      android:layout_width="match_parent"
```

Comment:
@aggarwalpulkit596 
When I change the width to `match_parent` the app crashes with following error:
<img width="1388" alt="Screenshot 2021-06-29 at 9 26 01 PM" src="https://user-images.githubusercontent.com/9396084/123830789-446bdd80-d921-11eb-9436-699d006dbbfc.png">

Is there any specific reason for this?

Reply:
@aggarwalpulkit596 Nevermind. Reverting this back to `wrap_content` does work for this PR correctly.